src/library/VUmeter.js

```diff
@@ -0,0 +1,76 @@
+import FIFOArray from './FIFOarray.js';
+
+/**
+ * This is a VUmeter Library for creating VUmeter
+ * @param {String} canvasId The HTML id where waveform is located.
+ * @param {Number} minDecibel The minimum lower bond of decible
+ *     If the decible is lower than this value, it cosider as slient.
+ */
+class VUmeter {
+  constructor(canvasId, misDecible) {
+    this.canvas = document.querySelector(canvasId);
+    this.canvasContext = this.canvas.getContext('2d');
+    this.width = this.canvas.width;
+    this.height = this.canvas.height;
+    this.currentX = 0;
+    this.previousY = this.height / 2;
+    // Due to the calculation and graph convinience,
+    // we set it to positive
+    this.minDisplayDecible = Math.abs(misDecible);
+    this.fifoArray = new FIFOArray(8);
```

Comment:
If FIFO is only for 8 items, do we really need a separate class for this? What's the reason behind of the FIFO class?

Also FIFO is only need for the min value. xThen we can easily compare 1 by 1 and keep the minimum value. Am I missing something?